.gatling/example.package.conf

```diff
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
```

Comment:
I guess the url is not yet valid because based on PR?

Reply:
Yep, it will be valid after documentation is published